sonaranalyzer-dotnet/src/Tests/SonarAnalyzer.UnitTest/Helpers/SonarAnalysisContextTest.cs

```diff
@@ -47,20 +48,25 @@ private class TestSetup
         });
 
         [TestMethod]
-        public void SonarAnalysis_NoIssueReported()
+        public void SonarAnalysis_NoIssueReportedIfAnalysisIsDisabled()

```

Comment:
Changed the test name slightly to make it clear it the test relates to the _suppress analysis_ delegate rather than the _suppress reporting_ delegate.